src/Microsoft.Azure.WebJobs.Host/Bindings/BindingFactory.cs

```diff
@@ -101,11 +101,11 @@ public IBindingProvider BindToGenericValueProvider<TAttribute>(Func<TAttribute,
         /// Create a binding provider for binding a parameter to an <see cref="IAsyncCollector{TMEssage}"/>. 
         /// Use the <see cref="IConverterManager"/> to convert form the user's parameter type to the TMessage type. 
         /// </summary>
-        /// <typeparam name="TAttribute">type of binding attribute on the user's parameter.</typeparam>
+        /// <typeparam name="TAttribute">Type of binding attribute on the user's parameter.</typeparam>
         /// <typeparam name="TMessage">'core type' for the IAsyncCollector.</typeparam>
-        /// <param name="buildFromAttribute">function to allocate the collector object given a resolved instance of the attribute.</param>
+        /// <param name="buildFromAttribute">Function to allocate the collector object given a resolved instance of the attribute.</param>
         /// <param name="buildParameterDescriptor">An optional function to create a specific ParameterDescriptor object for the dashboard. If missing, a default ParameterDescriptor is created. </param>
-        /// <param name="postResolveHook">an advanced hook for translating the attribute. </param>
+        /// <param name="postResolveHook">An advanced hook for translating the attribute. </param>
         /// <returns>A binding provider that applies these semantics.</returns>
         public IBindingProvider BindToAsyncCollector<TAttribute, TMessage>(
             Func<TAttribute, IAsyncCollector<TMessage>> buildFromAttribute, 
```

Comment:
Will you be working on getting AsyncCollector into this pattern next? I'd rather only have to absorb these changes once in Extensions.